packages/bpk-component-graphic-promotion/src/BpkGraphicPromo.js

```diff
@@ -109,11 +103,10 @@ const BpkGraphicPromo = (props: Props) => {
 
   const getTextClasses = (baseClass: string) =>
     getClassName(baseClass, `${baseClass}--${textAlign}`);
-
   return (
     <BpkCard
       className={cardClasses}
-      style={{ backgroundImage: `url(${image})`, color: textColor }}
+      style={style}
```

Comment:
The recommended way would be via CSS, this is already possible with `className` as this property is exposed and when passed in gets applied to the styles of the `BpkCard`

So doing it with CSS classes and styles is how this is done

Reply:
As discussed, I don’t think we can achieve having multiple background images without using style, as it’s the style attribute that we’re able to inject the correct backgroundImages to be used at runtime, otherwise we’d just have static CSS classes with the same images. Marking as resolved.